test/fixtures/simple_example/main.tf

```diff
@@ -37,3 +37,35 @@ resource "local_file" "gce-keypair-pk" {
     sshKeys = "ubuntu:${tls_private_key.main.public_key_openssh}"
   }
 }
+
+resource "null_resource" "wait_for_server" {
+  triggers = {
+    server_instance_id = "${uuid()}"
+  }
+
+  provisioner "remote-exec" {
+    inline = "while ! [ -f /etc/profile.d/forseti_environment.sh ]; do sleep 10; done"
```

Comment:
Calling this out - this approach does work but is a bit crude. Is it worth extracting this to a script?

Reply:
I generally put this kind of stuff into a file resource if only to get automated shellcheck and such.

Regarding the logic, sleep 10 seems like a long time.  The stat call is cheap, could tighten this up to 1 second or shorter.

Finally, it's an infinite loop.  Suggest adding some upper bound and a log message to give the user some feedback about what's going on.

Reply:
Good point about shellcheck, resolved in c6faeb9606d3934b6f1addabbe97a1f9f75222d9